tests/test_config_consistency.py

```diff
@@ -71,3 +71,33 @@ def test_status_code_202(self):
         assert spans[0]["type"] == "web"
         assert spans[0]["meta"]["http.status_code"] == "202"
         assert spans[0]["error"] == 1
+
+
+@scenarios.tracing_config_nondefault
+@features.tracing_configuration_consistency
+class Test_Config_UnifiedServiceTagging_CustomService:
+    """ Verify behavior of http clients and distributed traces """
+
+    def setup_specified_service_name(self):
+        self.r = weblog.get("/")
+
+    def test_specified_service_name(self):
+        interfaces.library.assert_trace_exists(self.r)
+        spans = interfaces.agent.get_spans_list(self.r)
+        assert len(spans) == 1, "Agent received the incorrect amount of spans"
+        assert spans[0]["service"] == "service_test"
+
+
+@scenarios.default
+@features.tracing_configuration_consistency
+class Test_Config_UnifiedServiceTagging_Default:
+    """ Verify behavior of http clients and distributed traces """
+
+    def setup_default_service_name(self):
+        self.r = weblog.get("/")
+
+    def test_default_service_name(self):
+        interfaces.library.assert_trace_exists(self.r)
+        spans = interfaces.agent.get_spans_list(self.r)
+        assert len(spans) == 1, "Agent received the incorrect amount of spans"
+        assert spans[0]["service"] != "service_test"
```

Comment:
We set `DD_SERVICE` for all scenarios by default, to `DD_SERVICE=weblog`. So we probably need a single test class, in the defautl scenario, and check for service equals `weblog`?

Reply:
I think this is a good idea. @mtoffl01, should we update the RFC to match this, given that we know what the default value of `service` will be unless overridden?

Reply:
Hmm ideally the "default" condition would check what happens when DD_SERVICE is not set anywhere. @smola , is there a reasonable way to disable the DD_SERVICE=weblog setting?

Reply:
That would need to happen in an scenario other than the default one. You could set DD_SERVICE to empty in the de scenario definition.